File: Intermediate/Topic-003-Introduction-to-Arrays/Homework/Q3-Count-of-elements/Count-of-elements.c
/**
 * @input A : Integer array
 * @input n1 : Integer array's ( A ) length
 * 
 * @Output Integer
 */

 int maximum(int *A, int n1) {
     int max, idx;
     max = A[0];
     for (idx = 1; idx < n1; idx++) {
         if (A[idx] > max) {
             max = A[idx];
         }
     }
    return max;

 }
int solve(int* A, int n1) {

    int max = maximum(A, n1);
    int idx, count;
    for (idx = 0; idx < n1; idx++) {
        if (A[idx] < max) {
            count += 1;
        }
    }

    return count;
}

File: Intermediate/Topic-003-Introduction-to-Arrays/Homework/Q3-Count-of-elements/Count-of-elements.py
def maximum(A):
    max = A[0]

    for i in range(len(A)):
        if (A[i] > max):
            max = A[i]
    return max

class Solution:
    # @param A : list of integers
    # @return an integer
    def solve(self, A):
        max = maximum(A)
        count = 0
        for i in range (len(A)):
            if (A[i] < max):
                count += 1
        return count
